src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,24 +1,120 @@
 package core.basesyntax;
 
+import java.util.LinkedList;
+import java.util.List;
+
 /**
  * <p>Реалізувати свою HashMap, а саме методи `put(K key, V value)`, `getValue()` та `getSize()`.
  * Дотриматися основних вимог щодо реалізації мапи (initial capacity, load factor, resize...)
  * За бажанням можна реалізувати інші методи інтрефейсу Map.</p>
  */
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+
+    private LinkedList<Entry<K, V>>[] elementData;
+    private int size;
+    private int currentCapacity = DEFAULT_CAPACITY;
```

Comment:
do we need this variable?
it is the same as `elementData .length`

Reply:
Updated